pkg/webui/console/views/device-overview/index.js

```diff
@@ -43,14 +45,26 @@ const m = defineMessages({
   latestData: 'Latest data',
   rootKeys: 'Root keys',
   keysNotExposed: 'Keys are not exposed',
+  failedAccessOtherHostDeviceMessage:
```

Comment:
```suggestion
  failedAccessOtherClusterDevice:
```

Avoid suffixing i18n messages with `Message`, as this is redundant.